documentation/rom_simulation/0d-solver/install/readme.md

```diff
@@ -2,7 +2,9 @@
 
 There are three ways to install svZeroDSolver:
 
-1. Download an installer from [SimTK Simvascular Downloads](https://simtk.org/frs/?group_id=188). 
+1. Download an installer from [SimTK Simvascular Downloads](https://simtk.org/frs/?group_id=188). This will download an executable based on the most recent release.
+
```

Comment:
Either list all installing options in a list or place the options in separate paragraphs.